### PR TITLE
fix: calculate checksum correctly when downloading command in one chunk

### PR DESCRIPTION
**Pull request by qzb**
Calculating checksum was failing when tarball was downloaded in one
chunk.

### CODE REVIEW

**Review by gpolek — APPROVED** [review]

**Comment by bkielbasa** [comment]
Any test for it? :)

**Comment by qzb** [comment]
@bkielbasa As you can easily check there is no tests at all 😔 We need to address this issue before releasing v3.

File: internal/dependency/manager.go [hunk 1]
```diff
@@ -250,22 +250,20 @@ func downloadFile(url string, file io.Writer) (checksum string, err error) {
```
Comment (wopol) [inline internal/dependency/manager.go:250]: interesting implementation these three lines:
```
        writer := io.MultiWriter(file, hash)

	if _, err = io.Copy(writer, resp.Body); err != nil {
		return checksum, err
	}
```
Can i change this when i will implement progressbar #7. 

Will be something like this:
```
        writer := io.MultiWriter(file, hash, counter)
```

**Review by wopol — APPROVED** [review]

File: internal/dependency/manager.go (hunk 1 above)
Comment (qzb) [reply internal/dependency/manager.go:250]: By interesting, you meant not very good? 😅 Of course, you can, but if you want I can fix it right now.

Comment (wopol) [reply internal/dependency/manager.go:250]: You can change it in this PR. In my opinion this is better and much more simple solution.

**Review by Mikoj — COMMENTED** [review]
error handling ?

**Pull request merged**